app/views/alchemy/admin/elements/_element.html.erb

```diff
@@ -23,15 +23,44 @@
         html: {id: "element_#{element.id}_form".html_safe, class: 'element-content'} do |f| %>
 
         <div id="element_<%= element.id %>_errors" class="element_errors"></div>
+
+        <!-- Ingredients -->
         <% if element.has_ingredients_defined? %>
           <div class="element-ingredient-editors">
-            <%= render element.ingredients, element_form: f %>
-          </div>
-        <% else %>
-          <div id="element_<%= element.id %>_content" class="element-content-editors">
-            <%= render element.contents %>
+            <%= render element.ingredients.select { |i| i.definition[:group].blank? }, element_form: f %>
```

Comment:
We try to avoid using `blank?` and `present?` whenever possible. Since users will not have an empty group name in their elements.yml file we can simply check for nil here.